### PR TITLE
[10.0] website_multi_theme: Backport for theme_customize_get fix

### PR DESCRIPTION
**Pull request by yelizariev**

### CODE REVIEW

**Comment by yelizariev** [comment]
One review is available in #442, because that PR includes this commit

File: website_multi_theme/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+from odoo.addons.website.controllers.main import Website
+
+
+class WebsiteMultiTheme(Website):
+
+    def _xml_id2key(self, xml_id):
+        view = request.env.ref(xml_id, raise_if_not_found=False)
+        if view:
+            return view.key
+        return None
```
Comment (yajo) [inline website_multi_theme/controllers/main.py:14]: I think it's better to `return xml_id` here...

File: website_multi_theme/controllers/main.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    @route()
+    def theme_customize_get(self, xml_ids):
+        """Extend in order to replace xml_id to key, because
+        view.xml_id is 'website_multi_theme.auto_view_ID_WEBSITE',
+        while client works with original IDs.
+
+        """
+        res = super(WebsiteMultiTheme, self).theme_customize_get(xml_ids)
+        res = [[
+            self._xml_id2key(xml_id) or xml_id
```
Comment (yajo) [inline website_multi_theme/controllers/main.py:25]: ... and remove the `or xml_id` here.

**Review by yajo — APPROVED** [review]

**Comment by yajo** [comment]
Don't forget to mention that this one is a backport from #436, I'd have merged instead of reviewing :laughing: 

Fast-tracking

**Pull request merged**